releasenotes-builder/src/main/java/com/github/checkstyle/globals/ReleaseNotesMessage.java

```diff
@@ -200,4 +213,14 @@ else if (ch == ' ') {
 
         return sb.toString();
     }
+
+    /**
+     * Escapes {@code @} symbols by placing them inside code block.
+     *
+     * @param str string to escape.
+     * @return escaped string.
+     */
+    private static String escapeAtSymbol(String str) {
+        return str.replaceAll("@", "`@`");
```

Comment:
I'm curious how not escaping `@return` would cause having a different contributor image every time. `@` itself isn't an HTML special character, is it? Not gonna lie, missing the whole point of solving this issue..

Reply:
This is github escaping, not HTML. In github you use `@` to target a specific person and give them a bump that something was said about them. Look at my comment  where I did `@romani` and how it is bolded, compared to me escaping in this sentence.

Reply:
Ohh, my brain didn't think of that late night yesterday. It makes complete sense now, thank you!

Reply:
I am ok to not escape them until we see them to be problatic